### ISSUE TITLE
It seems incapable of generalizing unlike other finetuning methods

### ISSUE THREAD

**Issue by ninjasaid2k** [OPPO-Mente-Lab/Subject-Diffusion#1]
<img width="1610" alt="human_viz 1ccd982a" src="https://github.com/OPPO-Mente-Lab/Subject-Diffusion/assets/99513716/1ab7be06-508c-4f53-8f09-e36937fe1a8f">

They all seem to have the same expression and don't seem like it's capable of generalizing to different facial expression and the faces don't match the styles in the other pictures.

**Comment by 1073521013** [comment]
This is a good question. Our limitations also point out that the biggest challenge for the current model is editing attributes and accessories within user-input images. I think the root cause of this issue lies in the limitations of our model framework and training data, i.e., each input reference image is single and independent, and the model has made more optimizations in terms of subject fidelity. There might be two possible directions for improving this issue in the future: one is the optimization of the data itself; the other is the enhancement of the framework design by adding feature decoupling.

**Comment by brianosaurus** [comment]
@ninjasaid2k I've had trouble getting the repo to train. Did you have one original image in your tarfile (and the mask)?
